Minestom/src/main/java/it/founderconnessi/banconnesso/Plugin.java

```diff
@@ -0,0 +1,62 @@
+package it.founderconnessi.banconnesso;
+
+import it.founderconnessi.lib.interfaces.ConfigInt;
+import it.founderconnessi.lib.interfaces.LoggerInt;
+import it.founderconnessi.lib.interfaces.PluginInt;
+import net.minestom.server.MinecraftServer;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Classe concreta che implementa l'intefaccia {@link PluginInt}.
+ */
+public class Plugin implements PluginInt {
+
+    /**
+     * Logger.
+     */
+    private final LoggerInt logger;
+
+    /**
+     * Config.
+     */
+    private final ConfigInt config;
+
+    /**
+     * Costruisce un oggetto di tipo {@link Plugin}.
+     * @param logger logger.
+     * @param config config.
+     */
+    public Plugin(@NotNull LoggerInt logger, @NotNull ConfigInt config) {
+        this.logger = logger;
+        this.config = config;
+    }
+
+    @Override
+    @NotNull
+    public String getPluginVersion() {
+        return BanConnesso.getInstance().getOrigin().getVersion();
+    }
+
+    @Override
+    @NotNull
+    public String getServerType() {
+        return "Spigot";
+    }
```

Comment:
Questo metodo dovrebbe ritornare la stringa "Minestom" in quanto viene usata per controllare gli aggiornamenti relativi a questo modulo. 
[Questa è la riga di codice dove viene usata](https://github.com/FounderConnessi/BanConnesso/blob/625e9630f1f6e120fe8fa47e5fba39a0940ebcd7/Library/src/main/java/it/founderconnessi/lib/utils/UpdateChecker.java#L61)